packages/compile-michelson/src/compileStrategy.ts

```diff
@@ -0,0 +1,61 @@
+import { CompilerResult, ICompileStrategy } from "@truffle/compile-common";
+import { Parser } from '@taquito/michel-codec';
+import { readFileSync } from "fs";
+import path from "path";
+
+import { version } from "../package.json";
+
+const compiler = {
```

Comment:
Huh, is this really right?  You're setting the compiler name and version, not to that of the compiler being invoked, but to that of the Truffle package?  Could you explain the reasoning behind this?

Reply:
There is no compiler being invoked. Michelson contracts are just parsed from plain Michelson to JSON Michelson using Taquito parser.

I'm open to suggestions for the name and version, but I just followed what was done on the AlphaTez branch for Michelson. https://github.com/trufflesuite/truffle/blob/alphaTez/packages/compile-michelson/src/index.ts

The Ligo compiler will have the version from the compiler that is invoked though.

Reply:
Oh, huh.  I wonder if there is some better way to indicate this though.  (Some fictional value for `name`, and maybe omitting the version entirely or something?)  @gnidan, do you have any thoughts on this...?